src/resources/Logs/Logs.ts

```diff
@@ -0,0 +1,29 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {
+    FacetsModel,
+    FacetsWithCountsModel,
+    GetFacetsParams,
+    GetLogsOrFacetsRequestBodyModel,
+    GetLogsParams,
+    LogsModel,
+} from './LogsInterfaces';
+
+export default class Logs extends Resource {
+    static baseUrl = `/logs/v1/organizations/${API.orgPlaceholder}`;
```

Comment:
not v2?

Reply:
there is a v2?

Reply:
Currently, we use v1 and in the swagger, we only have v1 for Logs I think.
https://bitbucket.org/coveord/admin-ui/src/25a198ad1b191edd03300574fbe1b48e0fb0b65b/packages/jsadmin-log-browser/src/Constants.ts#lines-3
![image](https://user-images.githubusercontent.com/52677246/100251923-4f830900-2f0d-11eb-86de-63833246ce3c.png)